src/modules/fw_pos_control_l1/FixedwingPositionControl.hpp

```diff
@@ -359,7 +366,7 @@ class FixedwingPositionControl final : public ModuleBase<FixedwingPositionContro
 					float pitch_min_rad, float pitch_max_rad,
 					float throttle_min, float throttle_max, float throttle_cruise,
 					bool climbout_mode, float climbout_pitch_min_rad,
-					uint8_t mode = tecs_status_s::TECS_MODE_NORMAL, float hgt_rate_sp = NAN);
```

Comment:
Just curious, why are we trying to remove the TECS modes as a parameter?

Reply:
Well so far they were only used for `set_detect_underspeed_enabled`, doing that with a simple bool seemed more appropriate to me. And we have an issue with responsibility with the tecs modes: currently they are set tecs-internal (e.g. if TECS detects an underspeed condition), so all that FW_POS_C can do is "recommend" a TECS mode.
We should prob figure out what these modes should exactly accomplish.